frontend/src/app/services/post-dimensions.service.ts

```diff
@@ -0,0 +1,20 @@
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
+import {HttpClient} from '@angular/common/http';
+import {NGXLogger} from 'ngx-logger';
+import {DummyParcelDimensions} from '../common/dummy-parcel-dimensions';
+import {ParcelSizeHolder} from '../common/parcel-size-holder';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PostDimensionsService {
```

Comment:
Please rename to CalculateParcelDimensionsService

Reply:
done